src/hb/ot_shaper_use_machine.rs

```diff
@@ -314,27 +292,18 @@ pub fn find_syllables(buffer: &mut hb_buffer_t) {
     }
 
     {
-        let mut _klen = 0;
         let mut _trans = 0;
         let mut _keys: i32 = 0;
-        let mut _acts: i32 = 0;
-        let mut _nacts = 0;
-        let mut __have = 0;
+        let mut _inds: i32 = 0;
+        let mut _ic = 0;
         '_resume: while (p != pe || p == eof) {
             '_again: while (true) {
-                _acts = (_use_syllable_machine_from_state_actions[(cs) as usize]) as i32;
-                _nacts = (_use_syllable_machine_actions[(_acts) as usize]) as u32;
-                _acts += 1;
-                while (_nacts > 0) {
-                    match (_use_syllable_machine_actions[(_acts) as usize]) {
-                        1 => {
-                            ts = p;
-                        }
-
-                        _ => {}
+                match (_use_syllable_machine_from_state_actions[(cs) as usize]) {
```

Comment:
Can you point me toward where these changes to `fn find_syllables()` are being ported from? I'm not saying you did it wrong, just that I'm having trouble locating the original changes to verify the port.

Reply:
https://github.com/harfbuzz/harfbuzz/commit/b6196986d7f17cd5d6aebec88b527726b1493a9c

This is auto-generated code from ragel-rust, so you won't find the difference in harfbuzz. I just updated the `.rl` file and then regenerated it.

Reply:
Right ho! Maybe next time around as you are porting if you can document the process you use including bits like this and the commands (even if it is just jotted notes that we can refactor into a guide later) not would it make reviewing easier it could help other contributors be able to jump in with updated ports.